src/solidClientHelpers/utils.js

```diff
@@ -230,3 +230,14 @@ export const ERROR_CODES = {
 export function isHTTPError(errorMessage, code) {
   return !!errorMessage.match(new RegExp(code));
 }
+
+export function sharedStart(...strings) {
```

Comment:
It returns the shared characters that all strings starts with, e.g. `sharedStart("bar", "baz", "bam")` will return "ba". I'm going to add a comment about this, since it's not apparent.

I needed to use it for `getPolicyUrl` where I want to find the corresponding path for the policy resource for a given resource. I compare the URI for the resource with the URI policies container, and generate a path based on this comparison.